perf/stability/mysql/README.md

```diff
@@ -0,0 +1,41 @@
+# MySQL Testing
+
+This tests ensures Istio working with MySQL, with or without mTLS enabled.
+
+The setup consists of two parts
+
+- A MySQL server.
+- A MySQL cmmand line client.
```

Comment:
nit: typo

Please also add a setup.sh that automates part or all of what is described in the README.

Reply:
Done.